models/spike_detector.h

```diff
@@ -103,18 +103,12 @@ class spike_detector : public Node
   spike_detector();
   spike_detector( const spike_detector& );
 
-  void set_has_proxies( const bool hp );
   bool
   has_proxies() const
   {
     return has_proxies_;
   }
-  bool
-  potential_global_receiver() const
-  {
-    return true;
-  }
-  void set_local_receiver( const bool lr );
+
   bool
   local_receiver() const
```

Comment:
I think this is needed, see, e.g., https://github.com/nest/nest-simulator/blob/master/nestkernel/connection_manager.cpp#L380